manifests/server/config.pp

```diff
@@ -163,6 +163,17 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if defined('$::servername') {
+      file { $::puppet::server::ssl_ca_crl:
+        ensure  => file,
+        owner   => $::puppet::server::user,
+        group   => $::puppet::server::group,
+        mode    => '0644',
+        content => file($::settings::cacrl, $::settings::hostcrl, '/dev/null'),
```

Comment:
$::settings was documented as early as: https://docs.puppet.com/puppet/3.5/

However playing in a docker container the $::settings var is functional all the way back to the oldest versions of facter + puppet:

```log
[root@5b5ecd7865de /]# puppet apply -e 'notify{"$::settings::server": }'
warning: Could not retrieve fact fqdn
notice: puppet
notice: /Stage[main]//Notify[puppet]/message: defined 'message' as 'puppet'
notice: Finished catalog run in 0.01 seconds
[root@5b5ecd7865de /]# facter --version
1.6.1
[root@5b5ecd7865de /]# puppet --version
2.6.9
```